ui/pages/confirm-transaction-base/transaction-alerts/transaction-alerts.js

```diff
@@ -111,9 +111,19 @@ const TransactionAlerts = ({
       )}
       {isNetworkBusy ? (
         <ActionableMessage
-          message={<I18nValue messageKey="networkIsBusy" />}
+          message={
```

Comment:
Thanks for adding this, I think I just copied from another place but didn't know how we normally do this 👍🏻